firefox-ios/Client/Frontend/PasswordGenerator/View/PasswordGeneratorHeaderView.swift

```diff
@@ -0,0 +1,65 @@
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this
+// file, You can obtain one at http://mozilla.org/MPL/2.0/
+
+ import Foundation
+ import Common
+
+final class PasswordGeneratorHeaderView: UIView, ThemeApplicable {
+    private enum UX {
+        static let headerIconLabelSpacing: CGFloat = 10
+        static let headerVerticalPadding: CGFloat = 8
+        static let headerImageHeight: CGFloat = 24
+    }
+
+    private let scaledHeaderImageSize = UIFontMetrics.default.scaledValue(for: UX.headerImageHeight)
```

Comment:
Great that the image is scaled. Is it's size updated as well when the user changes the dynamic font size while the password generator is shown?

Reply:
Not at the moment. On it!